<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NextMasterFormClass</class>
 <widget class="QDialog" name="NextMasterFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NextMasterForm</string>
  </property>
  <widget class="QLabel" name="lbNext">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>マスター基板をセットして読込を開始して下さい</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFinish">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>130</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>登録終了</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNone">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>基板をはずして読込を開始して下さい</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbMasterCountLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>枚目</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbMasterCount">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>99</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="prgImageReadState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStartNext">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>200</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>画像読込</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
